bonusy/inspirace/assets/kolotoc.py

```diff
@@ -0,0 +1,49 @@
+import sys
+import random
+import re
+
+from typing import List
+
+HIDDEN_CHAR = '▮'
+
+
+def update_text(hidden_text: List[str], matches: List, guess: str):
```

Comment:
Možná i návratovou hodnotu:

```suggestion
def update_text(hidden_text: List[str], matches: List, guess: str) -> str:
```